Main.java

```diff
@@ -1,5 +1,8 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("hello");
+        int[] numbers = {1, 2, 3};
+        for (int i = 0; i <= numbers.length; i++) {
+            System.out.println(numbers[i]);
```

Comment:
Potential division by zero error. Add a check to ensure 'y' is not zero before performing the division.